.github/workflows/packages-upload-wpk-images.yml

```diff
@@ -0,0 +1,44 @@
+name: Packages - Upload WPK images
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag name of the Docker image to be uploaded (Use developer to set branch name. Use auto to set src/VERSION content)'
+        required: false
+        default: auto
+jobs:
+  Upload-WPK-images:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Comment:
Similar to the packages, do we need to deploy all the containers? It would be beneficial to be able to select specific containers for deployment.

Reply:
done